### DISCUSSION TITLE
Using Monai label as traditional ML

### DISCUSSION THREAD

**Discussion by vedicsharma** [Project-MONAI/MONAILabel#1734] [Q&A]
I had a quick question: I am testing MONAI label radiology segmentation as active learning, but I also want to run it as traditional machine learning, just to compare with active learning. To do this, what I am doing is when I start a new session (I move the the old model at add/radiology/model to an archive folder so the model folder is empty and so the model is untrained when i start) and then for directories, I leave the labels and image files in their folders, and then when i start it on 3d slicer I don't add anymore images/annotations, just click train and thus train on what is already there. This will do what i want, ie train it essentially as traditional machine learning rather than active learning, correct? Thank you!

**Comment by diazandr3s** [comment]
Hi @vedicsharma,

That's right. If you don't add more annotated labels, active learning isn't involved, and MONAI Label will train on the existing annotated dataset.

Hope this helps!
